docs/content/routing/services/index.md

```diff
@@ -9,6 +9,34 @@ The `Services` are responsible for configuring how to reach the actual services
 
 ## Configuration Example
 
+??? example "Specify the port that traefik will use to connect to the service"
+
+    ```yaml tab="Docker"
+    # Sets the port traefik will use to connect to the container
+    labels:
+    - "traefik.http.services.my-service.loadbalancer.server.port=12345"
+    ```
+
+    ```toml tab="File (TOML)"
+    [http.services]
+      [http.services.my-service.loadBalancer]
+        [[http.services.my-service.loadBalancer.server]]
+          port = "12345"
+    ```
+
+    ```yaml tab="File (YAML)"
+    http:
+      services:
+        my-service:
+          loadbalancer:
+            server:
+              port=12345
+    ```
+
+!!! important
+    Traefik automatically uses the first port exposed in the container, this configuration will override that behavior.
+    This is useful for when traefik is connecting to the wrong port and you are receiveing a 502 gateway error.
+
```

Comment:
```suggestion
```

Reply:
Can you move this on the specialized "Docker Routing"  section please?  The reason is that the context here is not specific to Docker.